src/oAuth2.js

```diff
@@ -113,7 +113,8 @@ export class OAuth2 {
 
     return this.http.fetch(exchangeForTokenUrl, {
       method: 'post',
-      body: json(data),
+      headers: { 'Content-Type': 'application/json; charset=utf-8' },
+      body: JSON.stringify(data),
```

Comment:
Could we not add the Content-Type header directly in the fetch config?
By doing so, it's added for all fetch request, but I don't see any harm in that.
Quite confused about changing from json(data)--> JSON.stringify for the body. Are we sure this is ok for the other browsers?
@mbroadst , enjoy the holiday!